opts/opts_test.go

```diff
@@ -445,7 +446,16 @@ func TestValidateHost(t *testing.T) {
 		"tcp://invalid":      "Invalid bind address format: invalid",
 		"tcp://invalid:port": "Invalid bind address format: invalid:port",
 	}
+	var (
+		defaultHTTPHost = "tcp://127.0.0.1:2375"
+		defaultHOST     = "unix:///var/run/docker.sock"
+	)
+	if runtime.GOOS == "windows" {
+		defaultHOST = defaultHTTPHost
+	}
```

Comment:
Do you really mean the GOOS of where the test is running from? Should this not be dependent on the platform the daemon is running on instead? If so, it should be `if daemonPlatform == "windows"`